build.html

```diff
@@ -1,19 +1,37 @@
 <div class="chat-holder loading" data-chat-id="{{ id }}">
   <div class="top-header">
-    <div class="header-title" data-translate="widgets.chat.title">Messages</div>
+    <div class="header-title" data-translate="widgets.chat.title"></div>
```

Comment:
Is it perhaps better to leave a `&nbsp;` so that the layout doesn't shift down when the header eventually gets populated? Not sure if there's a css height already in place to avoid this.

Reply:
@squallstar I considered using `&nbsp;` too. I tested it to make sure this wasn't creating a layout issue.